test/services/cache/ExportsDeclarationBuilder.scala

```diff
@@ -26,175 +26,175 @@ import forms.declaration.additionaldeclarationtype.AdditionalDeclarationTypeSupp
 import forms.declaration.destinationCountries.DestinationCountries
 import forms.declaration.officeOfExit.OfficeOfExit
 import forms.{Choice, Ducr}
-import models.ExportsCacheModel
+import models.ExportsDeclaration
 import models.declaration.{DeclarationAdditionalActorsData, DeclarationHoldersData, Locations, TransportInformationContainerData}
 
 //noinspection ScalaStyle
-trait ExportsCacheModelBuilder {
+trait ExportsDeclarationBuilder {
 
   protected val DUCR = "5GB123456789000-123ABC456DEFIIIII"
   protected val LRN = "FG7676767889"
 
   private def uuid: String = UUID.randomUUID().toString
 
-  private val modelWithDefaults: ExportsCacheModel = models.ExportsCacheModel(
+  private val modelWithDefaults: ExportsDeclaration = ExportsDeclaration(
     sessionId = uuid,
     draftId = uuid,
     createdDateTime = LocalDateTime.of(2019, 1, 1, 0, 0, 0),
     updatedDateTime = LocalDateTime.of(2019, 2, 2, 0, 0, 0),
     choice = Choice.AllowedChoiceValues.StandardDec
   )
 
-  private type CacheModifier = ExportsCacheModel => ExportsCacheModel
+  private type ExportsDeclarationModifier = ExportsDeclaration => ExportsDeclaration
 
-  def aCacheModel(modifiers: CacheModifier*): ExportsCacheModel =
+  def aDeclaration(modifiers: ExportsDeclarationModifier*): ExportsDeclaration =
```

Comment:
Key change

Reply:
I'm still thinking about value of this `a` inside `aDeclaration`